horovod/torch/interface.h

```diff
@@ -0,0 +1,74 @@
+// Copyright 2018 Uber Technologies, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// =============================================================================
+
```

Comment:
missing include guard?

Reply:
This and the other interface.h file are actually just used by CFFI.  CFFI is doing its own parsing of a header file and it's pretty simplistic in terms of a subset of C that it understands.  Specifically, I don't believe it understands guards.